php/Dockerfile

```diff
@@ -3,6 +3,6 @@
 # GA updates: https://github.com/awsdocs/aws-doc-sdk-examples/issues/4131
 FROM composer/composer:latest
 COPY . /php/
-RUN cd /php/example_code/glue && \
-    composer install
+WORKDIR /php
```

Comment:
`WORKDIR` is not supported in GitHub Actions. Recommended approach would be to include the target path in the below `RUN find` command.

Reply:
Added the full path to the find command, but left the WORKDIR in for users who want to run the container directly.